pygubudesigner/propertieseditor.py

```diff
@@ -15,21 +15,23 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 # For further info, check  http://pygubu.web.here
-from __future__ import unicode_literals, print_function
+from __future__ import print_function, unicode_literals
+
 import logging
 
 try:
     import tkinter as tk
     import tkinter.ttk as ttk
-except:
+except BaseException:
```

Comment:
```suggestion
except ImportError:
```

I'm not spamming xdd